### PR TITLE
Add taskbar icon

### PR DESCRIPTION
**Pull request by mobilegmYT**
This is untested code but it *should* work.

### CODE REVIEW

File: src/mcpil.py [hunk 1]
```diff
@@ -106,6 +106,9 @@
 proxy_thread: threading.Thread = None
 proxy = Proxy()
 
+# Set icon in taskbar
+window.iconphoto(True, PhotoImage(file="/usr/share/pixmap/mcpil.png"));
```
Comment (nobody5050) [inline src/mcpil.py:110]: Maybe I’m completely wrong but shouldn’t this have a tilde like so?
`window.iconphoto(True, PhotoImage(file="~/usr/share/pixmap/mcpil.png"));`

Comment (mobilegmYT) [reply src/mcpil.py:110]: Tilde means home dir, /usr/share is a non home dir

Comment (nobody5050) [reply src/mcpil.py:110]: Oop I thought usr was part of home.

**Review by nobody5050 — APPROVED** [review]

**Comment by mobilegmYT** [comment]
Soooo like can you guys merge this?

**Comment by Alvarito050506** [comment]
~~No.~~ Did anyone test the changes yet?

**Comment by mobilegmYT** [comment]
> ~No.~ Did anyone test the changes yet?

Cool kids don't test their code because it works first try 😎

**Comment by mobilegmYT** [comment]
> ~No.~ Did anyone test the changes yet?

```
Traceback (most recent call last):
  File "/opt/mcpil/mcpil.py", line 110, in <module>
    window.iconphoto(True, PhotoImage(file="/usr/share/pixmap/mcpil.png"));
NameError: name 'window' is not defined
```

**Review by Alvarito050506 — APPROVED** [review]

**Pull request merged**